tests/reference/ast-prefix_string_02-3d530b2.stderr

```diff
@@ -0,0 +1,5 @@
+syntax error: Token '"Hello World"' (of type 'string') is unexpected here
+ --> tests/errors/prefix_string_02.py:7:10
+  |
+7 |  print(r "Hello World")
+  |          ^^^^^^^^^^^^^ 
```

Comment:
For comparison, this test passes in main, and in CPython 3.10.2 it gives:
```console
  File "/Users/ondrej/repos/lpython/a.py", line 7
    print(r "Hello World")
            ^^^^^^^^^^^^^
SyntaxError: invalid syntax
```
So this new LPython error is a good one.